Increase minimal required cmake version (#12022)
zephyrproject-rtos/zephyr: doc/getting_started/installation_linux.rst

   sudo pacman -S git cmake ninja gperf ccache dfu-util dtc wget \
       python-pip python-setuptools python-wheel xz file make

.. important::

>    Zephyr requires a very recent version of CMake. Make sure to read through

Reviewer: ```suggestion
   Zephyr requires a recent version of CMake. Read through
```

>    the rest of the section below in order to make sure the version you have

Reviewer: ```suggestion
   the rest of the section below to verify the version you have
```

   installed is recent enough to build Zephyr.

> CMake version 3.13.1 or higher is required. Check what version you have using

Reviewer: ```suggestion
CMake version 3.13.1 or higher is required. Check what version you have by using
```

``cmake --version``. If you have an older version, there are several ways

> of obtaining a more recent one, listed below.

Reviewer: ```suggestion
of obtaining a more recent one:
```

* Use ``pip``:

  .. code-block:: console

     pip3 install --user cmake

* Download and install from the pre-built binaries provided by the CMake
  project itself in the `CMake Downloads`_ page.
  For example, to install version 3.13.1 in :file:`~/bin/cmake`:

  .. code-block:: console

     mkdir $HOME/bin/cmake && cd $HOME/bin/cmake
     wget https://github.com/Kitware/CMake/releases/download/v3.13.1/cmake-3.13.1-Linux-x86_64.sh
     yes | sh cmake-3.13.1-Linux-x86_64.sh | cat
     echo "export PATH=$PWD/cmake-3.13.1-Linux-x86_64/bin:\$PATH" >> $HOME/.zephyrrc

* Check your distribution's beta or unstable release package library for an
  update.

.. note::

>    If you have installed a recent version of CMake using one of the appraoches

Reviewer: ```suggestion
   If you have installed a recent version of CMake using one of the approaches
```
